### DISCUSSION TITLE
Generate an image from the command line

### DISCUSSION THREAD

**Discussion by owl5053** [lllyasviel/Fooocus#3586] [Q&A]
Is it any way to set command line params for the image and output folder from the command line to generate an image from the command line and close the fooocus?
Something like this (example):
`.\python_embeded\python.exe -s Fooocus\entry_with_update.py -output="c:\mages" -prompt="some image" -negative="..." -checkpoints="juggernautXL_v8Rundiffusion.safetensors" -quit`

**Comment by luisamanett** [comment]
Hi, have you found a way? I'm interested too

**Reply by owl5053** [reply]
> Hi, have you found a way? I'm interested too

unfortunately not

**Comment by owl5053** [comment]
@luisamanett  https://tricks.one/post/get-fooocus-gradio-api-working-en/
